homeassistant/components/co2signal/manifest.json

```diff
@@ -1,9 +1,9 @@
 {
   "domain": "co2signal",
-  "name": "Electricity Maps",
```

Comment:
This change is incorrect and is reverting a change from current dev.

Reply:
Is gerevert naar originele waarde

Reply:
Please use English